.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+## Contributing guidelines
+
+Thank you for your interest in contributing to EvalAI! Here are a few pointers about how you can help.
+
+### Setting things up
+
+To set up the development environment, follow the instructions in README.
+
+### Finding something to work on
+
+The issue tracker of EvalAI a good place to start. If you find something that interests you, comment on the thread and we’ll help get you started.
+
+Alternatively, if you come across a new bug, please file a new issue and comment if you would like to be assigned. The existing issues are tagged with one or more labels, based on the part of the website it touches, its importance etc., that can help you in selecting one.
+
+If neither of these seem appealing, please post on our channel and we will help find you something else to work on.
+
+### Instructions to submit code
+
+Before you submit code, please talk to us via the issue tracker so we know you are working on it.
+
+Our central development branch is development. Coding is done on feature branches based off of development and merged into it once stable and reviewed. To submit code, follow these steps:
+
+1. Create a new branch off of development. Select a descriptive branch name.
+
+        git fetch upstream
+        git checkout master
+        git merge upstream/master
+        git checkout -b your-branch-name
+
+2. Commit and push code to your branch:
+
+    - Commits should be self-contained and contain a descriptive commit message.
+    - Please make sure your code is well-formatted and adheres to PEP8 conventions (for Python) and the airbnb style guide (for JavaScript). For others (Lua, prototxt etc.) please ensure that the code is well-formatted and the style consistent.
```

Comment:
You can remove PEP8 here, sincve it coontains only frontend. basically they need to run `ng lint` command to resolve the linting issue if there is any. They can also add extension for Angular 7 in text editor to fix it while coding only.